ecf/defs/prod00.def

```diff
@@ -2249,14 +2254,16 @@ suite prod00
   endfamily
   family gdas
     edit CYC '00'
-    edit ECF_FILES '/ecf/ecfnets/scripts/gdas_FV3'
-    edit PROJ 'GDAS-OPS'
+    edit PACKAGEHOME '/lfs/h2/emc/global/noscrub/Lin.Gan/git/feature-ops-wcoss2'
```

Comment:
This variable needs to be defined outside the `gdas` family as it is constant for the package.

Reply:
Explained in the previous similar comment. It should be inside family level.